examples/nextjs/src/components/content-types/activity.js

```diff
@@ -14,9 +15,10 @@ function Activity({ title, description, image, urlTitle }) {
             {image && (
                 <Image
                     className="w-full"
-                    src={`${process.env.NEXT_PUBLIC_DOTCMS_HOST}${
-                        image?.idPath || image
-                    }?language_id=${language?.id}`}
+                    loader={imageLoaderByConfig({
+                        language: language?.id ?? 1,
+                    })}
```

Comment:
Can't we make this global?

Reply:
Don't want to have to pass in every `<Image />`

Reply:
Done, the only down side is that in the footer, we cannot get the current `languageId` but is something that we can solve later.